docs/dev_guide/index.rst

```diff
@@ -35,6 +35,7 @@ This goes over the basics and has links to useful tutorials on git.
    contents/remote_data
    contents/config
    contents/funding
+   contents/ci
```

Comment:
This doesn't seem to have made it appear in the index page or menu? https://sunpy--6037.org.readthedocs.build/en/6037/dev_guide/index.html#

Reply:
That is really odd, I will debug this locally and hopefully push a fix.